distutils/command/install.py

```diff
@@ -400,7 +400,7 @@ def finalize_options(self):
                             'dist_version': self.distribution.get_version(),
                             'dist_fullname': self.distribution.get_fullname(),
                             'py_version': py_version,
-                            'py_version_short': '%d.%d' % sys.version_info[:2],
+                            'py_version_short': get_config_vars()["VERSION"],
```

Comment:
`distutils` is already migrating toward honoring `sysconfig` variables. What is `sysconfig.get_config_vars()['py_version_short']` in your environment?

Reply:
We return "3.8" for py_version_short since that seems to be implied by the name, but we modified install.py with this same change as in this PR. Perhaps a better solution is to change INSTALL_SCHEMES to use VERSION instead of py_version_short? I wasn't sure where these things were all used so I tried to make a smaller change which is why I did it this particular way.